WalletWasabi.Fluent/Extensions/DynamicDataExtensions.cs

```diff
@@ -76,4 +76,14 @@ public static class DynamicDataExtensions
 	{
 		return contents.Subscribe(list => sourceCache.Edit(updater => updater.Load(list)));
 	}
+
+	public static IObservable<IReadOnlyCollection<TObject>> ToCollectionStartWithEmpty<TObject, TKey>(this IObservable<IChangeSet<TObject, TKey>> changeSet) where TKey : notnull
+	{
+		return changeSet.ToCollection().StartWithEmpty();
+	}
+
+	public static IObservable<IReadOnlyCollection<TObject>> ToCollectionStartWithEmpty<TObject>(this IObservable<IChangeSet<TObject>> changeSet)
+	{
+		return changeSet.ToCollection().StartWithEmpty();
+	}
```

Comment:
How does it work? Does it always start with empty and then with the not empty? Or if it is not empty then it just starts with it?

Reply:
It always sends an "empty changeset" notification. For what I know, there's no way to add any conditional logic to send it or not.